test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -46,6 +46,37 @@ public void Add_adds_table()
         addedShape.Should().BeAssignableTo<ITable>();
     }
 
+    [Test]
+    public void Add_adds_picture_same_slide()
+    {
+        // Arrange
+        var pres = new Presentation(TestAsset("053_add_shapes.pptx"));
+        var copyingShape = pres.Slides[0].Shapes.GetByName("Picture")!;
+        var shapes = pres.Slides[0].Shapes;
+
+        // Act
+        shapes.Add(copyingShape);
+
+        // Assert
+        shapes.GetByName("Picture 1").Should().NotBeNull();
```

Comment:
Please add "assertion" `pres.Validate()` because of the test makes a side effect.

Reply:
13bc1cc8d2ef6d4d7cadf336e576acbf787d6807